packages/server/modules/multiregion/regionConfig.ts

```diff
@@ -33,19 +33,23 @@ const getMultiRegionConfig = async (): Promise<MultiRegionConfig> => {
     regions: {}
   })
 
-  if (isDevOrTestEnv() && !isMultiRegionEnabled()) {
-    return emptyReturn()
-  }
-
   if (!multiRegionConfig) {
     const relativePath = getMultiRegionConfigPath({ unsafe: isDevOrTestEnv() })
     if (!relativePath) return emptyReturn()
 
     const configPath = path.resolve(packageRoot, relativePath)
 
-    multiRegionConfig = await loadMultiRegionsConfig({
-      path: configPath
-    })
+    try {
+      multiRegionConfig = await loadMultiRegionsConfig({
```

Comment:
small unrelated fix - turning workspaces off w/ a multiregion DB doesn't work without this